airflow/www/extensions/init_jinja_globals.py

```diff
@@ -66,9 +66,13 @@ def prepare_jinja_globals():
             'airflow_version': airflow_version,
             'git_version': git_version,
             'k8s_or_k8scelery_executor': IS_K8S_OR_K8SCELERY_EXECUTOR,
-            'rest_api_enabled': conf.get('api', 'auth_backend') != 'airflow.api.auth.backend.deny_all',
+            'rest_api_enabled': False,
```

Comment:
Hmmm, this one isn't right.

This is used to control if the "Test Connection" button is enabled for the Connections form - see https://github.com/apache/airflow/pull/19792

So we probably want to test if `session` is in this list.

Reply:
Read the code Ash :faceplam: